src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -592,3 +592,24 @@ def test_office365(pyi_builder):
         SamlTokenProvider._prepare_request_from_template('RST2.xml', {})
         SamlTokenProvider._prepare_request_from_template('SAML.xml', {})
         """)
+
+
+@importorskip('spacy')
+def test_spacy(pyi_builder):
+    pyi_builder.test_source("""
+        import spacy
+        """)
+
+
+@importorskip('thinc')
+def test_thinc_banckends_numpy_ops(pyi_builder):
+    pyi_builder.test_source("""
+        from thinc.backends import numpy_ops
+        """)
+
+
+@importorskip('srsly.msgpack._packer')
+def test_srsly_msgpack_packer(pyi_builder):
+    pyi_builder.test_source("""
+        import srsly.msgpack._packer
```

Comment:
This one should probably test either `srsly.msgpack`or even just `srsly`. No need to go all the way down to the _packer.

Reply:
Changed to import srsly